### DISCUSSION TITLE
Welcome to blockchain-basics-cu Discussions!

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [Cyfrin/blockchain-basics-cu#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CodeAyoola** [comment]
hi, i've just completed the blockchain basics. wish me well

**Reply by samakpos33-code** [reply]
what did you understand about the updates options

**Reply by tibossoflagos** [reply]
Hi. Just started the blockchain basics.

**Reply by 0xAlpha01** [reply]
Congratulations! 🎉 Wish you all the best too. Keep going, you’re just getting started! 🚀

**Reply by tibossoflagos** [reply]
Thank you!

**Comment by RohitBorate** [comment]
Hi, I'm trying to do transaction which can be visible on [sepolia.etherscan.io](sepolia.etherscan.io). However, when I try to add SepoliaETH from faucets.chain.link , it is not allowing me to, can you suggest me some other way
![image](https://github.com/user-attachments/assets/4455e971-238c-49d1-a465-a6100fb7dea9)

**Reply by samakpos33-code** [reply]
did you finished up smart the blockchain practice project before getting too sepoliaETH

**Comment by thedevkansal** [comment]
Need help verifying my solidity contract on Zksync plugin...I have tried everything told in the video...my wallet is on zksync sepolia testnet...i am able to compile the code and even deploy it but can't toggle that verify contract button...don't know why
![Screenshot 2025-06-16 205658](https://github.com/user-attachments/assets/a800f667-440b-425e-bb20-53a0a6cd53fd)

**Reply by PatrickAlphaC** [reply]
Skip verification for now... You can deploy contracts to zksync using the regular EVM extension now... We will fix this part of the course soon.

**Comment by muhammadjanbahaccurate-collab** [comment]
Hi Guys I'm New

**Comment by Gidiehills** [comment]
Hello guys i'm new here and looking to become a blockchain security expert

**Comment by marycynthia2020** [comment]
Hey guys I need link to join the discord community. All the ones I saw has expired.

**Reply by PatrickAlphaC** [reply]
@marycynthia2020 where did you see these expired ones? We should fix...

Use this for now: https://discord.gg/sccXzKDVJp

**Comment by marycynthia2020** [comment]
On the website

**Comment by Dt-Musa** [comment]
Hey I'm Musa, just joined today, hoping to learn more

**Comment by yeab21-crypto** [comment]
Hi,my name is yeasira i like very very this course an i want to aprreciate the prepare this course ,and i have question ...what is smart contract i was trying to understand, but still i don't konw the what is the concept

**Reply by 0xAlpha01** [reply]
Base on my understanding of what **smart contract** means:
so, **Smart contract** is basically a program that runs on a **blockchain** and **automatically** carries out an agreement when certain conditions are met.
Think of it like a _vending machine_. You put in money, choose a drink, and the machine automatically gives you the drink if everything is correct. You don't need a person standing there to approve it.
Smart contracts work in a similar way. For example:: "If you send **0.1** ETH, the smart contract automatically sends you the **NFT (Non-Fungible Token)**." 
The rules are written in code, and the blockchain executes those rules automatically. This means you don't need a middleman to manually handle the transaction.
Smart contracts are commonly used in DeFi, NFTs, DAOs, and decentralized applications (dApps).

**Comment by yeab21-crypto** [comment]
OK

**Comment by Komalrawatt** [comment]
Hey I am Komal Rawat from India, I have recently started this course, the content is amazing and love the structured learning content

**Reply by 0xAlpha01** [reply]
Woo, that's awesome! 🎉 It's been 3 weeks since I started learning blockchain too, and it's been pretty easy to understand so far.
